profiles/seccomp/default.json

```diff
@@ -199,12 +200,17 @@
 				"link",
 				"linkat",
 				"listen",
+				"listmount",
 				"listxattr",
+				"listxattrat",
 				"llistxattr",
 				"_llseek",
 				"lremovexattr",
 				"lseek",
 				"lsetxattr",
+				"lsm_get_self_attr",
+				"lsm_list_modules",
+				"lsm_set_self_attrs",
```

Comment:
lsm_set_self_attr

Reply:
Are we sure this is a secure syscall?

Reply:
re security, you can already do what this syscall does via userspace procfs: `/proc/self/attr/`